internal/handler/Handler.go

```diff
@@ -66,36 +67,58 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.updateEmployee(w, r)
 		return
 	default:
-		fmt.Println("Error: Not Found!")
+		fmt.Println("error: not found")
 		http.NotFound(w, r)
 		return
 	}
 }
 
 func (h *Handler) getPositions(w http.ResponseWriter, r *http.Request) {
-	positions:=h.service.GetPositions()
+	query, err := url.ParseQuery(r.URL.RawQuery)
+	limit := query["limit"][0]
+	offset := query["offset"][0]
+	if err != nil{
+		http.Error(w, "error with server", http.StatusInternalServerError)
+		return
+	}
+	positions, err :=h.service.GetPositions(limit, offset)
+	if err != nil{
+		http.Error(w, "error with server", http.StatusInternalServerError)
+		return
```

Comment:
You have to check which error you get GetPositions(limit, offset) function. If it something like "position not found" you have to return 404 error, rather than 500